modules/console/pages/config/analytics.adoc

```diff
@@ -0,0 +1,64 @@
+= Redpanda Console Telemetry
+:description: Understand what telemetry Redpanda Console collects by default, how it is handled, and how to disable it.
+
+Redpanda Console collects telemetry (analytics) data to improve the product and user experience. This document explains what data is collected, how it is processed, and how you can disable telemetry if desired.
+
+== What is tracked by default
+
+When telemetry is enabled, which is the default behavior, Redpanda Console collects metadata to help improve product functionality and reliability. This data is sent securely to Redpanda.
+
+=== Redpanda Console metadata
+
+The following information is collected from the running instance of Redpanda Console:
+
+- Startup timestamp (when the Redpanda Console process starts)
+- Runtime UUID (a unique ID generated per instance)
+- Redpanda Console version and build timestamp
+- License hash, license type, and license organization
+- Kafka configuration hash (anonymized)
+- Whether Kafka is connected as `localhost`
+
+=== Redpanda cluster metadata
+
+The following information is collected from the connected Kafka cluster:
+
+- Cluster configuration details (excluding sensitive values)
+- Broker log directories
```

Comment:
This may be interpreted as the "data" that is stored on the brokers. Maybe reword to "Broker log dir statistics such as the size"?